corehq/apps/app_manager/static/app_manager/json/commcare-profile-settings.yaml

```diff
@@ -8,6 +8,15 @@
   since: "1.3"
   force: true
 
+- name: "Lazy Load Resources"
+  description: "How should Commcare load the the multimedia files in the application."
```

Comment:
It isn't clear what the default is. Can you rephrase to something like "Load multimedia files lazily"?

Reply:
Done